src/clr-core/internal/utils/dom.ts

```diff
@@ -18,6 +20,13 @@ export function getElementWidthUnless(element: HTMLElement, unless: boolean) {
   return '';
 }
 
+export function isHTMLElement(el: any) {
+  if (el) {
+    return el instanceof HTMLElement;
```

Comment:
Just a minor thing. We could simplify this further to...

`return !!el && el instanceof HTMLElement`